R/data_color.R

```diff
@@ -388,7 +431,7 @@ html_color <- function(colors, alpha = NULL) {
   }
 
   # Create a color matrix with an `alpha` column
```

Comment:
The whole colors[!is_rgba] -> #RRGGBB, should be extracted out into a separate function